File: remove-element.cpp
// Answer to https://leetcode.com/problems/remove-element
// Amal George M
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        auto itrEnd = std::remove(nums.begin(), nums.end(), val);
        auto pos = std::distance(nums.begin(),itrEnd);
        return (static_cast<int> (pos));
    }
};

File: remove_element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        for i in range(len(nums)):
            try:
                nums.remove(val)
            except:
                return len(nums)   
